config/spells/timed.json

```diff
@@ -859,6 +860,7 @@
 			"noneOf" : {
 				"bonus.MIND_IMMUNITY" : "normal",
 				"bonus.NON_LIVING" : "normal",
+				"bonus.MECHANICAL" : "normal",
 				"bonus.SIEGE_WEAPON" : "absolute",
```

Comment:
Why siege weapon and mechanical get different immunity type?

Reply:
In original H3 Implosion spell cannot be cast on catapult. Otherwise defending hero could just wipe attacker`s catapult on turn 1 (10 SP and Expert Earth are enough) and leave an opponent with heavily limited options to continue castle siege